### PR TITLE
List of meetings implemented

### PR DESCRIPTION
**Pull request by pierrecf**
Summary
Created a UI to display a list of one or more meetings, showcasing all meetings available.

Details
Implemented the following changes to accomplish the user story:

Implemented UI components to render the list of meetings.
Designed the layout to accommodate one or more meetings effectively.
Integrated functionality to fetch and display all meetings dynamically.

### CODE REVIEW

File: src/app/modules/app/pages/meetings/meetings.component.ts [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
 
 @Component({
   selector: 'app-meetings',
   templateUrl: './meetings.component.html',
   styleUrls: ['./meetings.component.sass'],
 })
-export class MeetingsComponent {}
+export class MeetingsComponent implements OnInit {
+  headers: string[] = MEETING_HEADERS;
+  data: Meeting[] = [];
```
Comment (AJN-jalaU) [inline src/app/modules/app/pages/meetings/meetings.component.ts]: Maybe we should avoid using "data" word. Everything can be "data".
Here "data" could be meetings or something similar

File: src/app/modules/app/pages/meetings/meetings.component.ts [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+    this.isLoading = true;
+    this.meetingService.getAll().subscribe({
+      next: this.handleSuccess.bind(this),
+      error: this.handleError.bind(this),
+    });
+  }
+
+  private handleSuccess(data: Meeting[]): void {
+    console.log(data);
+    this.data = data;
```
Comment (AJN-jalaU) [inline src/app/modules/app/pages/meetings/meetings.component.ts]: Remove Console log
Maybe we should avoid using "data" word. Everything can be "data".
Here "data" could be meetings or something similar

**Review by AJN-jalaU — APPROVED** [review]

**Review by Adrian-Mendoza-Jala — COMMENTED** [review]
Do you have an image of how it looks?

**Comment by AJN-jalaU** [comment]
> Do you have an image of how it looks?

we can use the PR template:
https://tree.taiga.io/project/marcel-morales-klockan/wiki/template-to-pull-request

**Comment by pierrecf** [comment]
Screenshot 2024-02-02 at 4 29 32 PM
Yeah this is how it looks, i just fixed the issues metioned byt @AndresJavierNazzari

**Comment by pierrecf** [comment]
Summary
Created a UI to display a list of one or more meetings, showcasing all meetings available.

Details
Implemented the following changes to accomplish the user story:

Implemented UI components to render the list of meetings.
Designed the layout to accommodate one or more meetings effectively.
Integrated functionality to fetch and display all meetings dynamically.
Screenshot 2024-02-02 at 4 29 32 PM

**Pull request merged**